hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -2261,9 +2260,12 @@ public List<OzoneFileStatus> listStatus(OmKeyArgs args, boolean recursive,
         ListStatusRequest.newBuilder()
             .setKeyArgs(keyArgs)
             .setRecursive(recursive)
-            .setStartKey(startKey)
             .setNumEntries(numEntries);
 
+    if (startKey != null) {
+      listStatusRequestBuilder.setStartKey(startKey);
```

Comment:
`startKey` is required for `ListStatusRequest`.  So now we get the following instead of `NullPointerException`:

```
com.google.protobuf.UninitializedMessageException: Message missing required fields: startKey
	at com.google.protobuf.AbstractMessage$Builder.newUninitializedMessageException(AbstractMessage.java:770)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$ListStatusRequest$Builder.build(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ozone.om.protocolPB.OzoneManagerProtocolClientSideTranslatorPB.listStatus(OzoneManagerProtocolClientSideTranslatorPB.java:2274)
```